tests/integration/security/util/cert/cert.go

```diff
@@ -102,6 +102,8 @@ func CreateCASecret(ctx resource.Context) error {
 		},
 	}
 
+	// Just clean any existing cacerts.
```

Comment:
Can also check for errors.IsNotFound and instead do an update. I think we do this in other places is why I mention it

Reply:
Changed to use IsAlreadyExists